_docs/eFormidling/installasjon/eformidling_auto_deploy.md

```diff
@@ -0,0 +1,204 @@
+
+*Som standard er desse endepunkta eksponerte lokalt i integrasjonspunktet*
+
+---
+
+# Installasjon
+Før deploymanager<tbd> kan automatisk laste ned ny versjon og oppdatere ditt køyrande integrasjonspunkt må den setjast opp ved å konfigurere properties og velge ein katalog det skal køyre frå. 
+
+Det er anbefalt (minst konfigurasjon) å køyre både integrasjonspunkt.jar og deploymanager.jar<tbd> frå samme katalog, om ønska køyre frå forskjellige katalog [sjå her](https://github.com/felleslosninger/efm-deploy-manager#running-deploymanager-and-integrasjonspunkt-from-different-folders).
+
+
+1. Legg inn jar-fila og ```deploymanager-local.properties``` i ønska katalog.
+2. Sett opp naudsynte konfigurasjonar i ```deploymanager-local.properties```. Sjå under.
+3. [Laste ned Digdir sin offentlege nøkkel](/resources/eformidling/public_keys/eformidling-key.asc) og lagre valgt katalog.
+
+## Konfigurere properties fil
+Åpne ```deploymanager-local.properties``` i katalogen du skal køyre ```.jar``` fila frå sett inn følgande properties.
+
+```java
+# Replace hosts and ports of URL with the location
+# of your integrasjonspunkt.
+deploymanager.integrasjonspunkt.baseURL=http://localhost:9093
+
+# E-mail is optional. Please specify these properties 
+# to receive e-mails when the deploy-manager updates the integrasjonspunkt-application.
+deploymanager.mail.recipient=someone@yourdomain.no
+deploymanager.mail.from=noreply@yourdomain.no
+
+spring.mail.host=smtp.yourdomain.no
+spring.mail.port=<set-your-port-here>
+
+# Digitaliseringsdirektoratet public key paths. i.e: file:keyname.gpg.
+deploymanager.verification.publicKeyPaths[0]=file:eformidling-key.gpg
+```
+*[Last ned properties-fila her](/resources/eformidling/deploymanager-local.properties)*
+
+### Setje tidspunkt for oppdatering
+*Valgfritt*
+
+Ein kan setje tidspunkt for kortid applikasjonen vil forsøke å oppdatere integrasjonspunktet om ein ikkje ynskjer å benytte standard-verdiane. 
+
+Her er nokre døme som viser korleis ein kan styre tidspunkt for oppdatering.
+
+```java
+#Standard verdi: Sjekkar etter oppdatering mandag-fredag kl 05:30, 19:30 og 21:30.
+deploymanager.schedulerCronExpression=0 30 5,19,21 * * MON-FRI
+
+#Sjekkar etter oppdatering kvar dag kl 06:00.
+deploymanager.schedulerCronExpression=0 0 6 * * ?
+
+#Sjekkar etter oppdatering kvar dag kl 23:15.
+deploymanager.schedulerCronExpression=0 15 23 ? * *
+
+#Sjekkar etter oppdatering kvar laurdag og søndag kl 12:00.
+deploymanager.schedulerCronExpression=0 0 12 ? * SAT,SUN
+
+#Sjekkar etter oppdatering kvart tredje minutt kvar time.
+deploymanager.schedulerCronExpression=0 0,3,6,9,12,15,18,21,24,27,30,33,36,39,42,45,48,51,54,57 * ? * *
+```
+
+## Verifisere sertifikatet
+Når Digitaliseringsdirektoratet publiserer eit nytt integrasjonspunkt vil dette være signert med vår privat nøkkel. For å verifisere denne signaturen kan du laste ned vår offentlege nøkkel og sjekke om fingeravtrykket på signaturen er likt som nøkkelen.
```

Comment:
Kan det gå tydligare fram at det viktigaste er å verifisere signatur for deploy-manager og eventuelt integrasjonspunktet også dersom ein vel å laste ned dette manuelt ved installasjon. Om ein allereie har køyrande integrasjonspunkt med versjon x.x.x eller tidligare så er dette ikkje signert

Reply:
Gode innspel, legg inn.